pathfinding_service.rst

```diff
@@ -18,6 +18,16 @@ Assumptions
 * Every pathfinding service is responsible for a single token network. Pathfinding services are scaled on process level to handle multiple token networks.
 
 
+High-Level-Description
+======================
+A node can request a list of possible paths from start point to endpoint for a given transfer value.
+The get_paths method implements canonical Dijkstra algorithm to return a given number of paths for a mediated transfers of a given value. The design regards the Raiden network as unidirectional weighted graph, where the default weights (and therefore the primary constraint of the optimization) are the fees of each channel. Additionally we applied two heuristics to quantify desirable properties of the resulting graphs:
```

Comment:
- implements *the*/*a* canonical Dijkstra
- for a mediated transfer (remove the s)
- as *an* unidirectional
- properties of the resulting graph (should be just one, right?)